android/src/main/java/vn/luongvo/kmm/survey/android/ui/screens/survey/views/SurveyQuestion.kt

```diff
@@ -27,15 +29,15 @@ fun SurveyQuestion(
         modifier = Modifier.fillMaxSize()
     ) {
         DimmedImageBackground(
-            // TODO fetch survey detail https://github.com/luongvo/kmm-survey/issues/23
-            imageUrl = "https://dhdbhh0jsld0o.cloudfront.net/m/1ea51560991bcb7d00d0_l"
+            imageUrl = question.coverImageUrl
         )
 
         CloseButton(
             modifier = Modifier
                 .statusBarsPadding()
                 .align(Alignment.TopEnd)
-                .padding(vertical = dimensions.paddingMedium, horizontal = dimensions.paddingSmall),
+                .padding(vertical = dimensions.paddingMedium, horizontal = dimensions.paddingSmall)
+                .semantics { this.contentDescription = SurveyCloseButton + index },
```

Comment:
I think we can omit `this`

Reply:
Fixed in https://github.com/luongvo/kmm-survey/pull/69/commits/de24b9cb9ebd308b493eb3fcbc29ead96ed05e67